save-backend/src/main/kotlin/org/cqfn/save/backend/service/TestService.kt

```diff
@@ -79,7 +77,9 @@ class TestService {
         val execution = agent.execution
         val lock = locks.computeIfAbsent(execution.id!!) { Any() }
         return synchronized(lock) {
-            val testExecutions = getTestExecutionsBatchByExecutionIdAndUpdateStatus(execution)
+            val testExecutions = transactionTemplate.execute {
+                getTestExecutionsBatchByExecutionIdAndUpdateStatus(execution)
+            }!!
```

Comment:
If no executions found we will return null here, no?

Reply:
Should be an empty list